macros/materializations/model.sql

```diff
@@ -69,9 +69,11 @@
         ml_config=ml_config,
         labels=raw_labels
     ) }}
+    {%- if ml_config['MODEL_TYPE'] != 'TENSORFLOW'-%}
```

Comment:
We should probably support users specifying options in lower case. I, for one, would tend to do the following:
```
{{
    config( 
        materialized='model',
        ml_config={ 
            'model_type': 'tensorflow',
            ...
       } 
    ) 
}}
```
Something like this should work:
```
{%- if ml_config.get('MODEL_TYPE', ml_config.get('model_type', '')).lower() != 'tensorflow' -%}
```

Reply:
Thanks for reviewing the change and for improving the PR.
Your change seems to do the trick for both lowercase and uppercase, I have pushed one more commit with that change.